### PR TITLE
Fix typos in cloud v2 create

### PR DESCRIPTION
**Pull request by daxmc99**

### CODE REVIEW

File: content/departments/cloud/technical-docs/v2.0/creation_process.md [hunk 1]
```diff
@@ -16,13 +16,21 @@ cd cloud
...
+    - [Init deployment artifacts - GCP Project](#init-deployment-artifacts---gcp-project)
+    - [Init deployment artifacts - Infrastructure](#init-deployment-artifacts---infrastructure)
+    - [Init deployment artifacts - K8S](#init-deployment-artifacts---k8s)
+    - [Deploy application](#deploy-application)
+    - [Commit your changes](#commit-your-changes)
+  - [Troubleshooting](#troubleshooting)
+    - [PVC is stuck at `Pending`](#pvc-is-stuck-at-pending)
+    - [How do I check my Sourcegraph deployment rollout progress?](#how-do-i-check-my-sourcegraph-deployment-rollout-progress)
+    - [I am unable to apply the terraform module due to permission error.](#i-am-unable-to-apply-the-terraform-module-due-to-permission-error)
+    - [Any other questions?](#any-other-questions)
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/v2.0/creation_process.md]: can we revert the change here?

this section is meant to provide an overview of the creation process instead of ToC of the entire page, and there's already a built-in ToC on the right. also, I found it really hard to maintain the ToC by hand.

CleanShot 2022-09-28 at 09 45 41

**Review by michaellzc — APPROVED** [review]

File: content/departments/cloud/technical-docs/v2.0/creation_process.md (hunk 1 above)
Comment (daxmc99) [reply content/departments/cloud/technical-docs/v2.0/creation_process.md]: Ah this was an automated update, reverted
